finagle-redis/src/main/scala/com/twitter/finagle/Redis.scala

```diff
@@ -72,6 +73,8 @@ object Redis extends Client[Command, Reply] with RedisRichClient {
     private val stack: Stack[ServiceFactory[Command, Reply]] = StackClient.newStack
       .insertBefore(DefaultPool.Role, RedisPool.module)
       .insertAfter(StackClient.Role.prepConn, ConnectionInitCommand.module)
+      .replace(StackClient.Role.protoTracing, RedisTracingFilter.module)
+      .insertAfter(RedisLoggingFilter.role, RedisLoggingFilter.module)
```

Comment:
Does the `RedisLoggingFilter.role` exist in the client stack already? I believe using `insertAfter` here will not put it into the stack, since the role doesn't exists. Maybe before the tracing filter?

Reply:
Ah yes, the role does not currently exist in the stack. Docs from `insertAfter` say: `If no elements match the role, then an unmodified stack is returned.` So looks like this is currently won't add the logging filter. I'll change it to insert before the tracing filter.